### ISSUE TITLE
Command Line `--login` Provides a Bad Link

### ISSUE THREAD

**Issue by ryanjob42** [mzmine/mzmine#2044]
### Basic information

* My operating system and version: Rocky Linux 8.8
* My mzmine version: 4.2.0 (I'm using the Linux portable zip file)

### What happened

I am running MZMine on an HPC cluster. When I try to log in by running `mzmine --login`, I get one of two issues depending on how I SSH into the cluster.

If I SSH with X11 forwarding enabled, I get the following exception message:

```
2024-08-21 12:21:02 SEVERE io.github.mzmine.javafx.mvci.FxUpdateTask run Unhandled exception java.lang.UnsupportedOperationException: The BROWSE action is not supported on the current platform! while processing task User login
java.lang.UnsupportedOperationException: The BROWSE action is not supported on the current platform!
        at java.desktop/java.awt.Desktop.checkActionSupport(Desktop.java:381)
        at java.desktop/java.awt.Desktop.browse(Desktop.java:531)
        at io.mzio.users.client.DesktopProvider.browse(DesktopProvider.java:37)
        at io.mzio.users.client.CustomKeycloakInstalled.loginDesktop(CustomKeycloakInstalled.java:263)
        at io.mzio.users.client.CustomKeycloakInstalled.login(CustomKeycloakInstalled.java:221)
        at io.mzio.users.client.UserLoginService.login(UserLoginService.java:89)
        at io.mzio.users.client.UserLoginService.downloadUserFile(UserLoginService.java:149)
        at io.mzio.users.gui.fx.UserLoginTask.process(UserLoginTask.java:77)
        at io.github.mzmine.javafx.mvci.FxUpdateTask.run(FxUpdateTask.java:52)
        at io.mzio.users.gui.fx.UsersController.loginOrRegisterConsoleBlocking(UsersController.java:72)
        at io.github.mzmine.main.MZmineCore.main(MZmineCore.java:254)

2024-08-21 12:21:02 SEVERE io.github.mzmine.main.MZmineCore main Error during mzmine start up
java.lang.IllegalStateException: Desktop was not initialized. Requires mzmineDesktop
        at io.github.mzmine.main.MZmineCore.getDesktop(MZmineCore.java:409)
        at io.github.mzmine.main.MZmineCore.main(MZmineCore.java:257)
```

If I SSH without X11 forwarding, it will print a link that I am supposed to use to log in. However, if I follow that link, I get the page shown below.

![image](https://github.com/user-attachments/assets/a44a4dbc-85e5-4c48-89e0-1ee94219d287)

For now, I can work around the issue by logging in from a different system, copying the `.mzuser` file, and updating the `.mzconfig` file (per #1827). However, having the console print the correct link would be preferable.

**Comment by robinschmid** [comment]
We are testing out different solutions considering security.
Currently one of the easiest ways is to download the user locally, push it to the server into the user directory in user/.mzmine/users then it will be picked up. 
Otherwise just push the user to any directory and define the user as CLI argument:

Something like this on  Windows:
-u "D:\user\testuser.mzuser"

https://mzmine.github.io/mzmine_documentation/commandline_tool.html#argument-table

**Comment by ryanjob42** [comment]
Thank you for the information! I'll direct other users on my cluster to use the CLI argument for now until the `--login` argument is working.

**Comment by robinschmid** [comment]
Next version will be released soon mzmine 4.3.0. 
-login requires web browser support on the machine
-login-console was now added for pure console login with username and password input
